src/AccordionCollapse.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import Collapse from './Collapse';
+import AccordionContext from './AccordionContext';
+
+const propTypes = {
+  /**
+   * A key that corresponds to the toggler that triggers this collapse's expand or collapse.
+   */
+  eventKey: PropTypes.string.isRequired,
+};
+
+const AccordionCollapse = React.forwardRef(
+  ({ children, eventKey, ...props }, ref) => {
+    const context = useContext(AccordionContext);
+
+    return (
+      <Collapse in={context.activeEventKey === eventKey} {...props}>
+        <div ref={ref}>{children}</div>
```

Comment:
does this need to render this extra `<div>`? the `<Card.Body>` or whatever is going to render a `<div>` itself, no?

Reply:
The extra `<div>` seems to be there due to an issue with `Collapse`, if you don't render `children` in a wrapped element. For example, If I were to change the implementation to:
```jsx
<Collapse in={context.activeEventKey === eventKey} {...props}>
  {children}
</Collapse>
```
Then this error is thrown:
```
    Error: Uncaught TypeError: Cannot read property 'props' of undefined (webpack:///src/Collapse.js:213 <- test/index.js:136762)
        at Object.invokeGuardedCallbackDev (webpack:///node_modules/react-dom/cjs/react-dom.development.js:199 <- test/index.js:63588:16)
        at invokeGuardedCallback (webpack:///node_modules/react-dom/cjs/react-dom.development.js:256 <- test/index.js:63645:31)
        at replayUnitOfWork (webpack:///node_modules/react-dom/cjs/react-dom.development.js:18532 <- test/index.js:81921:5)
        at renderRoot (webpack:///node_modules/react-dom/cjs/react-dom.development.js:19422 <- test/index.js:82811:13)
        at performWorkOnRoot (webpack:///node_modules/react-dom/cjs/react-dom.development.js:20296 <- test/index.js:83685:7)
        at performWork (webpack:///node_modules/react-dom/cjs/react-dom.development.js:20208 <- test/index.js:83597:7)
        at performSyncWork (webpack:///node_modules/react-dom/cjs/react-dom.development.js:20182 <- test/index.js:83571:3)
        at requestWork (webpack:///node_modules/react-dom/cjs/react-dom.development.js:20051 <- test/index.js:83440:5)
        at scheduleWork (webpack:///node_modules/react-dom/cjs/react-dom.development.js:19865 <- test/index.js:83254:5)
```
Which seems to be somehow correlated to this code block in `Collapse`:
```jsx
className: classNames(
  className,
  children.props.className, // <-- This line may be causing the issue
  collapseStyles[state],
  this.getDimension() === 'width' && 'width',
),
```

Reply:
Well, `children` needs to be exactly one React element. Perhaps type `children` as `React.Element`, and perhaps do `React.Children.only(children)`?